learning/intermediate/session-4.js

```diff
@@ -45,8 +45,24 @@ const person8 = null
 // IF they CAN drink then you will console log out there {toastMessage} AND THEN return TRUE
 // IF they can NOT drink, do nothing and return FALSE
 // keep in mind that it's POSSIBLE for the object to be empty or undefined, and therefore the properties {age} and {toastMessage} may NOT be defined
-function canPersonDrink() {
+function canPersonDrink(person) {
   // fill in the function here
+
+  const checkAge = person.age
+  const checkToastMsg = person.toastMessage
+  const validateAge = () => {
+    checkAge >= 21 && console.log(checkToastMsg)
+    return true
+  }
+
+  const falseReturn = () => {
+    return false
+  }
+
+  typeof person === 'object' && checkAge
+  checkAge === true && checkToastMsg
+  checkAge === true && checkToastMsg === true && validateAge
+  validateAge === false && falseReturn
```

Comment:
I don't understand the purpose of any of these lines. They do nothing. And you're referencing `validateAge` again but nothing will happen because you're not actually calling it.